Source/WebCore/platform/VideoDecoder.cpp

```diff
@@ -51,9 +55,10 @@ void VideoDecoder::create(const String& codecName, const Config& config, CreateC
 }
 
 
-void VideoDecoder::createLocalDecoder(const String& codecName, const Config&, CreateCallback&& callback, OutputCallback&& outputCallback, PostTaskCallback&& postCallback)
+void VideoDecoder::createLocalDecoder(const String& codecName, const Config& config, CreateCallback&& callback, OutputCallback&& outputCallback, PostTaskCallback&& postCallback)
 {
 #if USE(LIBWEBRTC)
+    UNUSED_PARAM(config);
```

Comment:
Why isn't this in the `#else` clause?

Reply:
It should be here AND in the `#else` indeed. Thanks!